bears/python/PyImportSortBear.py

```diff
@@ -182,7 +184,31 @@ def run(self, filename, file,
 
         if new_file != tuple(file):
             diff = Diff.from_string_arrays(file, new_file)
+            all_imports = []
```

Comment:
There should at least be one newline between line 184 and line 214 - this is a bit unreadable IMO.